configs/raspberrypi_config

```diff
@@ -1,28 +1,36 @@
 # Binaries generated with the following script:
 #    https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/build-uboot-rpi.sh
 
-# Raspberry Pi does not support GRUB bootloader integration, fallback to U-boot.
-MENDER_GRUB_EFI_INTEGRATION=n
-
-# Nothing to copy
-MENDER_COPY_BOOT_GAP=n
-
-# 4MB alignment
-MENDER_PARTITION_ALIGNMENT="4194304"
 
 RASPBERRYPI_BINARIES="${RASPBERRYPI_CONFIG}-2020.01.tar.gz"
 RASPBERRYPI_BINARIES_URL="${MENDER_STORAGE_URL}/mender-convert/uboot/raspberrypi/${RASPBERRYPI_BINARIES}"
 
+RASPBERRYPI_OS_REQUIRED_ID="raspbian" # historical distro name of the Raspberry Pi OS Debian derivative
```

Comment:
I have a bookworm image 32 bits with "debian" in os_release 
![image](https://github.com/user-attachments/assets/c64efd66-2a4f-40bd-8758-7001e6428f02)